plugins/outputs/groundwork/groundwork_test.go

```diff
@@ -50,8 +52,17 @@ func TestWriteWithDefaults(t *testing.T) {
 		require.NoError(t, err)
 	}))
 
+	testLogger := func() telegraf.Logger {
+		if err := logger.SetupLogging(&logger.Config{Debug: true}); err != nil {
+			return nil
+		}
+		l := new(testutil.Logger)
+		log.Logger = NewLogger(l).WithGroup("tcg.sdk")
+		return l
+	}()
+
```

Comment:
This should not be necessary! Please make sure the tests pass without this! Using `testutil.Logger` should be sufficient as otherwise there is something fishy going on... Same for the instances below.

Reply:
done